src/main/java/org/cactoos/list/Immutable.java

```diff
@@ -198,7 +207,9 @@ public int lastIndexOf(final Object item) {
 
     @Override
     public List<T> subList(final int start, final int end) {
-        return this.list.subList(start, end);
+        return new Immutable<>(
```

Comment:
@iakunin Just question. Why not just using: `new Immutable(this.list.subList(start, end))` ? What is the benefit of your approach?

Reply:
@fanifieiev you're absolutely right: there is no benefit of this approach. It could be simplified.

Done.